src/ReplicatedStorage/Client/InstanceComponents/Misc/Sprint.lua

```diff
@@ -0,0 +1,63 @@
+local Players = game:GetService("Players")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local ContextActionService = game:GetService("ContextActionService")
+local Trove = require(ReplicatedStorage.Packages.Trove)
+local Comm = require(ReplicatedStorage.Packages.Comm)
+
+local Sprint = {}
+Sprint.__index = Sprint
+
+function Sprint.new(character: Model)
+	local clientComm = Comm.ClientComm.new(character, false, "Sprint")
+	local self = setmetatable({}, Sprint)
+
+	self._character = character
```

Comment:
This should be `self.Instance = character` for consistency. Every component should *probably* be expected to expose an `Instance` field publicly.